assignments/objects.js

```diff
@@ -19,46 +19,130 @@ const example = {
 
 // Write your intern objects here:
 
+const mitzi = {
+  "id": 1,
+  "name": "Mitzi",
+  "email": "mmelloy0@psu.edu",
+  "gender": "F"
+};
+
+const kennan = {
+  "id": 2,
+  "name": "Kennan",
+  "email": "kdiben1@tinypic.com",
+  "gender": "M"
+};
+
+const keven = {
+  "id": 3,
+  "name": "Keven",
+  "email": "kmummery2@wikimedia.org",
+  "gender": "M"
+};
+
+const gaunnie = {
+  "id": 4,
+  "name": "Gaunnie",
+  "email": "gmartinson3@illinois.edu",
+  "gender": "M"
+};
+
+const antonietta = {
+  "id": 5,
+  "name": "Antonietta",
+  "email": "adaine5@samsung.com",
+  "gender": "F"
+};
+
+
+
+
 
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
+console.log( '\n ==== Challenge 2: Reading Object Data ==== ' );
 
 // Mitzi's name
-
+console.log( mitzi[ "name" ] );
```

Comment:
Consider not having spaces between object key and the brackets. 
```suggestion
console.log( mitzi["name"] );
```